env.example

```diff
@@ -16,6 +16,7 @@ FEED_URL=https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List
 
 # REDIS_URL specifies Redis server info
```

Comment:
Let's add another comment here discussing what to do if you're connecting via docker-compose, and need to use `redis://redis:6379`

Reply:
In this commit, the docker-compose actually overrides this variable, so we don't need to change it.